a/staging/src/kubevirt.io/api/core/v1/deepcopy_generated.go b/staging/src/kubevirt.io/api/core/v1/deepcopy_generated.go
index 16f004695..425940926 100644
--- a/staging/src/kubevirt.io/api/core/v1/deepcopy_generated.go
+++ b/staging/src/kubevirt.io/api/core/v1/deepcopy_generated.go
@@ -609,6 +609,22 @@ func (in *ConfigMapVolumeSource) DeepCopy() *ConfigMapVolumeSource {
 	return out
 }
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *ContainerDiskInfo) DeepCopyInto(out *ContainerDiskInfo) {
+	*out = *in
+	return
+}
+
+// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new ContainerDiskInfo.
+func (in *ContainerDiskInfo) DeepCopy() *ContainerDiskInfo {
+	if in == nil {
+		return nil
+	}
+	out := new(ContainerDiskInfo)
+	in.DeepCopyInto(out)
+	return out
+}
+
 // DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
 func (in *ContainerDiskSource) DeepCopyInto(out *ContainerDiskSource) {
 	*out = *in
@@ -1940,6 +1956,22 @@ func (in *I6300ESBWatchdog) DeepCopy() *I6300ESBWatchdog {
 	return out
 }
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *InitrdInfo) DeepCopyInto(out *InitrdInfo) {
+	*out = *in
+	return
+}
+
+// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new InitrdInfo.
+func (in *InitrdInfo) DeepCopy() *InitrdInfo {
+	if in == nil {
+		return nil
+	}
+	out := new(InitrdInfo)
+	in.DeepCopyInto(out)
+	return out
+}
+
 // DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
 func (in *Input) DeepCopyInto(out *Input) {
 	*out = *in
@@ -2251,6 +2283,48 @@ func (in *KernelBootContainer) DeepCopy() *KernelBootContainer {
 	return out
 }
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *KernelBootStatus) DeepCopyInto(out *KernelBootStatus) {
+	*out = *in
+	if in.KernelInfo != nil {
+		in, out := &in.KernelInfo, &out.KernelInfo
+		*out = new(KernelInfo)
+		**out = **in
+	}
+	if in.InitrdInfo != nil {
+		in, out := &in.InitrdInfo, &out.InitrdInfo
+		*out = new(InitrdInfo)
+		**out = **in
+	}
+	return
+}
+
+// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new KernelBootStatus.
+func (in *KernelBootStatus) DeepCopy() *KernelBootStatus {
+	if in == nil {
+		return nil
+	}
+	out := new(KernelBootStatus)
+	in.DeepCopyInto(out)
+	return out
+}
+
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *KernelInfo) DeepCopyInto(out *KernelInfo) {
+	*out = *in
+	return
+}
+
+// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new KernelInfo.
+func (in *KernelInfo) DeepCopy() *KernelInfo {
+	if in == nil {
+		return nil
+	}
+	out := new(KernelInfo)
+	in.DeepCopyInto(out)
+	return out
+}
+
 // DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
 func (in *KubeVirt) DeepCopyInto(out *KubeVirt) {
 	*out = *in
@@ -5447,6 +5521,11 @@ func (in *VirtualMachineInstanceStatus) DeepCopyInto(out *VirtualMachineInstance
 			(*in)[i].DeepCopyInto(&(*out)[i])
 		}
 	}
+	if in.KernelBootStatus != nil {
+		in, out := &in.KernelBootStatus, &out.KernelBootStatus
+		*out = new(KernelBootStatus)
+		(*in).DeepCopyInto(*out)
+	}
 	if in.TopologyHints != nil {
 		in, out := &in.TopologyHints, &out.TopologyHints
 		*out = new(TopologyHints)
@@ -5922,6 +6001,11 @@ func (in *VolumeStatus) DeepCopyInto(out *VolumeStatus) {
 		*out = new(DomainMemoryDumpInfo)
 		(*in).DeepCopyInto(*out)
 	}
+	if in.ContainerDiskVolume != nil {
+		in, out := &in.ContainerDiskVolume, &out.ContainerDiskVolume
+		*out = new(ContainerDiskInfo)
+		**out = **in
+	}
 	return
 }
 
